test/browser/support/TestHarness.js

```diff
@@ -0,0 +1,173 @@
+const BAV_COP_REQUEST_SENT = require("../support/BAV_COP_REQUEST_SENT_SCHEMA.json");
+const BAV_COP_RESPONSE_RECEIVED = require("../support/BAV_COP_RESPONSE_RECEIVED_SCHEMA.json");
+const BAV_CRI_END = require("../support/BAV_CRI_END_SCHEMA.json");
+const BAV_CRI_START = require("../support/BAV_CRI_START_SCHEMA.json");
+const BAV_CRI_VC_ISSUED = require("../support/BAV_CRI_VC_ISSUED_SCHEMA.json");
+const axios = require("axios");
+const aws4Interceptor = require("aws4-axios").aws4Interceptor;
+const { unmarshall } = require("@aws-sdk/util-dynamodb");
+const { fromNodeProviderChain } = require("@aws-sdk/credential-providers");
+const { XMLParser } = require("fast-xml-parser");
+const { expect } = require("@playwright/test");
+
+module.exports = class TestHarness {
+  constructor() {
+    this.HARNESS_API_INSTANCE = axios.create({
+      baseURL: process.env["TEST_HARNESS_URL"],
+    });
+    const customCredentialsProvider = {
+      getCredentials: fromNodeProviderChain({
+        timeout: 1000,
+        maxRetries: 0,
+      }),
+    };
+    const awsSigv4Interceptor = aws4Interceptor({
+      options: {
+        region: "eu-west-2",
+        service: "execute-api",
+      },
+      credentials: customCredentialsProvider,
+    });
+
+    this.HARNESS_API_INSTANCE.interceptors.request.use(awsSigv4Interceptor);
+  }
+
+  async getSession(sessionId) {
```

Comment:
This also doesn't look like it's being used?

Reply:
Removed